Cubical/Foundations/GroupoidLaws.agda

```diff
@@ -0,0 +1,95 @@
+{-
+
+This file proves the higher groupoid structure of types
+
+-}
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Foundations.GroupoidLaws where
+
+open import Cubical.Core.Primitives
+open import Cubical.Core.Prelude
+
+private
+  variable
+    ℓ : Level
+    A : Set ℓ
+
+-- some useful notation (move to prelude?)
+
+_·_ :  {x y z : A} → (x ≡ y) → (y ≡ z) → (x ≡ z)
+x≡y · y≡z = compPath x≡y y≡z 
+
+infixr 30 _·_
+
+_⁻¹ : {x y : A} → (x ≡ y) → (y ≡ x)
+x≡y ⁻¹ = sym x≡y
+
+-- homogeneous groupoid laws
+
+symRefl : (x : A) →
```

Comment:
How about renaming refl to 1?

Reply:
> I found a really cool unicode symbol instead, 𝟭.

It's a library, a weird unicode like 𝟭 will make people unable to use it.

I think what @mortberg meant was rename `symRefl` as `sym1`?

Reply:
yeah, let's wait for his response :)

Reply:
Ah I see, let's not do this then.